src/pocketmine/utils/MainLogger.php

```diff
@@ -180,7 +180,7 @@ public function shutdown(){
 	protected function send($message, $level, $prefix, $color){
 		$now = time();
 
-		$thread = \Thread::getCurrentThread();
+		$thread = Thread::getCurrentThread();
```

Comment:
This change is for better IDE autocompletion. You might want to revert this change.